starcheck/zodiacal_brightness.py

```diff
@@ -0,0 +1,58 @@
+import os
+import numpy as np
+from astropy.table import Table
+from Ska.Sun import position
+
+TABLE = os.path.join(os.path.dirname(__file__), 'data', 'table_17.csv')
+RESPONSIVITY = 0.0108 # (e-/sec/SI unit zodiacal flux)
+
+def equat2eclip(ra, dec):
```

Comment:
I would have preferred to use ecliptic coordinates from a more modern version of astropy (support introduced in 1.1 I think), but more modern versions of astropy require updated numpy and that seemed bigger than we needed right now in Py2 Ska.  If we swap over to Py3 Ska for starcheck relatively quickly, this routine could just be cut.

Tested against the Perl port of the IDL we were already using, but I'll include some more notes about that in this PR.